portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,33 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  /**private List<String> quotes;
+
+  @Override
+  public void init() {
```

Comment:
A proper PR shouldn't contain debugging information and should clean up commented out code though.